device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/BatchMessages.java

```diff
@@ -0,0 +1,30 @@
+package com.microsoft.azure.sdk.iot.device;
+
+import java.util.List;
+
+public class BatchMessages extends Message
```

Comment:
A javadoc at the class level would be helpful. It should explain what it is, and link it to the relevant device client method by using "{@ link \<classname\>#\<methodName\>}"

Reply:
I can't do the spacing right on that string above due to github, but autocomplete should help you out here